PowerShell Scanners/Get Bitlocker information/Scan Profile.xml

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<AdminArsenal.Export Code="PDQInventory" Name="PDQ Inventory" Version="19.0.40.0" MinimumVersion="19.0">
+  <ScanProfile>
+    <Collections type="list" />
+    <Scanners type="list">
+      <Scanner>
+        <ModifiedDate>2020-05-04T11:19:45.0000000+01:00</ModifiedDate>
+        <Name>Bitlocker Information</Name>
+        <Script>if (-Not(Get-Command Get-BitlockerVolume -ErrorAction SilentlyContinue)) {

```

Comment:
This scanner export isn't referencing the .ps1 file that your added. Can you change this so it uses the FileName field instead?

Reply:
Hi Bryan, I've made the suggested changes to the scan profile.